README.md

```diff
@@ -48,7 +48,7 @@ GitHub Action for automatically syncing LeetCode submissions to a GitHub reposit
 
        steps:
          - name: Sync
-           uses: joshcai/leetcode-sync@v1.6
+           uses: AlyHG/leetcode-sync@master
```

Comment:
Could you revert this line? I'll create a new release after this is in, and bump the version number up.